website/content/docs/upgrading/upgrade-specific.mdx

```diff
@@ -150,7 +150,7 @@ be done on most earlier versions of Consul+Envoy in advance of the Consul
 
 As an example, here's a Zipkin integration
 [before](https://github.com/hashicorp/consul/blob/v1.9.5/test/integration/connect/envoy/case-zipkin/service_s2.hcl)
-and [after](https://github.com/hashicorp/consul/blob/71d45a34601423abdfc0a64d44c6a55cf88fa2fc/test/integration/connect/envoy/case-zipkin/service_s2.hcl)
+and [after](https://github.com/hashicorp/consul/blob/71d45a34601423abdfc0a64d44c6a55cf88fa2fc/test/integration/connect/envoy/case-zipkin/service_s2.hcl).
```

Comment:
```suggestion
* [xDS v2 configuration](https://github.com/hashicorp/consul/blob/71d45a34601423abdfc0a64d44c6a55cf88fa2fc/test/integration/connect/envoy/case-zipkin/service_s2.hcl).
```
I think the larger issue is that it isn't clear what the examples are intended to illustrate.